edenai_apis/apis/nyckel/nyckel_api.py

```diff
@@ -301,14 +301,16 @@ def __create_label_if_no_exists(
 
         url = f"https://www.nyckel.com/v1/functions/{project_id}/labels"
         payload = {"name": label_name, "description": label_description}
+        response = self._session.post(url, json=payload)
```

Comment:
here, maybe catch boths: 
```python
try:
    response = self._session.post(url, json=payload)
    original_response = response.json()
except:
    raise ProviderException(...)
because I think we may have also an exception just from the post like a timeout error